src/jit/compiler.h

```diff
@@ -661,23 +701,23 @@ class  LclVarDsc
     bool                IsFloatRegType() const
                         {
                             return
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
                                 lvIsHfaRegArg ||
 #endif
                                 isFloatRegType(lvType);
                         }
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
It is needed because the member variables lvHfaTypeIsFloat is conditionally defined